src/main/java/wolox/training/models/dto/BookDTO.java

```diff
@@ -0,0 +1,52 @@
+package wolox.training.models.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.List;
+import java.util.Objects;
+import lombok.Data;
+import org.springframework.util.CollectionUtils;
+
+@Data
+public class BookDTO {
+
+    private String isbn;
+    private String title;
+    private String subtitle;
+    private List<DataName>  publishers;
+    @JsonProperty("publish_date")
```

Comment:
You can use this property in the application.properties file `spring.jackson.property-naming-strategy=SNAKE_CASE` to automatically map to snake case without using `@JsonProperty` 😄  
Just a suggestion though! You don't need to change it now :)